app/uploaders/avatar_uploader.rb

```diff
@@ -0,0 +1,61 @@
+# encoding: utf-8
+
+class AvatarUploader < CarrierWave::Uploader::Base
+
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  include CarrierWave::MiniMagick
+
+  # Choose what kind of storage to use for this uploader:
+  #if Rails.env.production? || Rails.env.development?
+    storage :fog
+  #else
+   # storage :file
+  #end
+  # storage :fog
+  def extension_white_list
+    %w(jpg jpeg gif png)
+  end
+
+  # Override the directory where uploaded files will be stored.
+  # This is a sensible default for uploaders that are meant to be mounted:
+  def store_dir
+    "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
+  end
+
+
+
+  # Provide a default URL as a default if there hasn't been a file uploaded:
+  # def default_url
+  #   # For Rails 3.1+ asset pipeline compatibility:
+  #   # ActionController::Base.helpers.asset_path("fallback/" + [version_name, "default.png"].compact.join('_'))
+  #
+  #   "/images/fallback/" + [version_name, "default.png"].compact.join('_')
+  # end
+
+  # Process files as they are uploaded:
+  #process :scale => [200, 300]
+
+  # def scale(width, height)
+  #   @width = 200
+  #   @height = 300
+  # end
+
+  # Create different versions of your uploaded files:
+  #version :thumb do
```

Comment:
Incorrect indentation detected (column 2 instead of 4).<br>Missing space after #.